index.html

```diff
@@ -1,4 +1,5 @@
-<!-- This develop branch will be used for all changes, we will raise pull requeast to merge all changes back tio main-->
+<!-- This develop branch will be used for all changes, we will raise pull requeast to merge all changes back-->
+<!-- this is my second comment -->
```

Comment:
next time, use capital letter for your comment

Reply:
`<!--NEW COMMENT-->`

Reply:
okay